src/lyra/core/types.py

```diff
@@ -91,10 +110,15 @@ def resolve_type_annotation(annotation):
             return FloatLyraType()
         elif annotation.id == 'str':
             return StringLyraType()
+        elif annotation.id == 'ndarray':
+            return NdarrayLyraType(None)
 
     if isinstance(annotation, ast.Subscript):
         if annotation.value.id == 'List':
             value = resolve_type_annotation(annotation.slice.value)
             return ListLyraType(value)
+#        elif annotation.value.id == 'Ndarray':
+#            value = resolve_type_annotation(annotation.slice.value)
+#            return NdarrayLyraType(value)
```

Comment:
This was an attempt to support element types, right? Add a comment that justifies why you leave this commented code here or remove it entirely.